modules/cmp/impl/clusters/import_cluster_test.go

```diff
@@ -0,0 +1,134 @@
1rZXktZGF0YTogTFMwdExTMUNSVWRKVGlCU1UwRWdVRkpKVmtGVVJTQkxSVmt0TFMwdExRcE5TVWxGYjJkSlFrRkJTME5CVVVWQmJYcEdSakEwWlZGMk9WaFVOMFZ5Wm5OWFYzbzFMM00yV25kaVFWcHVaRXB4ZUZGTGVWSjVObWRpY0dOa1RrRkRDbGhUVkdwWlMwRkxlVmxNYkdWNVdqVk5VVEUyVWpsVGJrNWxWMjFZY0ZGeFVISkJhMlV3ZGpsYVFuVnNaa05RYUhKc2JtdzNSMm95TmxZM2NYb3hNVWdLYUd4a1VWTkxja1l5T0V4Sk1rbGlOM3BPYzB4aE0xaHdOakJVSzBOeFlUbHNWMHd4U0dvNGRUaFpSRGRNVjNwMlR6UkRLMlkzVFZRek5YQnZOVk4wTVFwQ2NHeFZSMUkzY1RnM1MyRk5ZM0pSUldadldYWTVXREowV0hVelYwdE9SV012U0dKVk1IVkJVazlVVmtVeU1qWXhPVFZrTjNFM1VXTk5WRFJ3VVRJNUNqaFdPRXhQUW5NeFYyTkZWbXRuT1ZwcFVFVTNVa1JUUWpSSE5VVnFVak5xTHpOalExUTVUbWhMYzNkT1dHbEVOM2hRYkV0RWIyZDRRMmxqYkhOTWNITUtVV3d6ZEhGaFVYQk5OMXBHWm5kMVlrODRLeTk0TlhCdlVFaDFOMWhDZDBaRGVURm5SMUZKUkVGUlFVSkJiMGxDUVVkamVVWlpaRFpRZFU5Q1ZWbEdkZ3B5UmpKMlJHcHNNVVZxYms5QlNWSnBWVUZrUm5SWlNsUTJlVlV5WjJsS1pUWjVUall3ZDA4cmFVVXpSa1E0YldKS1RERmpaeTlLWkRCeE5rSlFaV0p2Q2xOVmMxVklOelJ3U1hBMk9YbzBXRVEyYkRkbloyOWFOSEZoT0ROSldtRjNTbFUyVFc5MVRXWlJOa0pHV0hZMk1HazFNa2RFYjNKMmFIQlBaV1JtYzNBS1RYbHRlQ3QxUzFwRGVVMXRURVl2YUdKNFl6TklabGRYV1N0SGMwb3pkMHQyTlhVMFJIaE5hVUZDWld0S2JTODNNVlpRTURsMVZrdGphRFJWU0RoNVpncFJVMjVEU1dKd2RtTXlLMmxsWlhvclozVTJRa1J6UTJoRFltOUpZVUZPUTNRMVFrRklXVmwxY0hVMk1sRkZkVmMwWVU5QlpreEtZV1E0VDBwTk1WUndDaTh3T0dvMGRtVkZPWHBPUzNKNmN6VjRaVmhDTUhwQ1lqRk5SVlpHVFVrek9YbHFOMDFMYzBReVNrNW9iaXRFYm1ocFFqSlFkemhqUVhVNE9GVXJieklLVEZST1kxTklNRU5uV1VWQmVVVnZiRVJQV2pGTmJqRTFWa2REYWtSSGQxQTFlVGt2U2twemFtZDNjMFUzZW14bFkzZEdOMUZFTlRjMVlWVTFiR1pTUndwb1lXdGlSamh2ZEdoWFRIVkZPVzhyTUZac1JTOUVTbXRJYUVsd0t6QllTVzRyTUVaa1IydG1OMDFNUVZGaVlYazNhMWw0Tm10d2VWUnFRVEZxTms5SENqTm1RbTVEVFRGM1VUbGlSblpyYUM5VFltRm9NVTFCVEdFeU5VTnFPVzByYWxoalVUTldkR3hKU1c5d1ZWTkhOa3hDYWtkclF6aERaMWxGUVhoc2RuVUtlSEpDVURsb2FHcHRRVFpTWjBNeWFXWlFRWEZsUkZFMGJVNU1PV0l5YVdGMVNHMWhNV2MyVFZCTU1HTndTRzVFVm01TlFtTlBOelZHVDFoTlpuVjFiZ3BWWmtneU1XeERjVmhqWjNOa1dVdDVSVkZCUjJ4b2JISklOVUpEYVV4aVdGUnJaVUpZVlhWWmJtb3pVMG96ZEU5TlptZ3dTbWxOVkZGWWJXMXZVRUp6Q25GUFJ6RXdNVmhFZG05WFlVbGxWbFl5VUcwMVpsSjRhVGh2U3pJemVuTm5jM0pQVTJWcVkwTm5XVUkwZGxKSFYzUkVjVVZLUmxkSGQzWmhibGd5UlRZS01sVjNTVGhwYVROTU1scGlSemRrVUVaME5rbDJUMGxPWW5OMlNFWk5iVXRsTkZkcmEzSkdZMVJ0V0RoMU4zcFJhMDFNWjFWVVVTOVhSek5OVm5GWU5RcHRibGRIVm01VWVsVm1aRFUxWW5ZMWJsbEZjbTV3VGtob1VrcFZjelY0ZVc5RFFrMXJlVTlTUldGRFJUZzRhMll3Tms4NGVqaFNVMm81V1RaU01saDVDbVUwZEVWR1dIUm9ORlZtUlRZNVRVTkZSRVp4Tm5kTFFtZEhjRzl3VDFnellsWXpVMXBhVFV4MFlXZDVWWFZzT1VkUWRGVTBSWFY2Vnk5MGJ5OVNPR0lLVGxOSVFuVm9PVmhRT0dSSUwzRTRaMnRPVWxwcWEzUlZiakpuZVRKS1lWRkhPU3RMWkVkd1RUSlFWRTVCVDAwemVGWkRLMGxCV2xaeFEzVmFSV3hXVHdvM1MyNHdlRTVIVjNwdFkxcHpZa0ZETTJGUllubDBaV292SzNnd1RGbGFTRloyWjJoSFVtNWhjQzh5Ynk5dE1tZFJXVFZLVEZKa01WbGpOMDA1UmxOS0NrZHViblpCYjBkQlYzbFVWbGhFVlhGUVdXaFNVRlZ1VTBNdmRsUTRNV05MVW5GVmJHdEllRVYyTkVJeWVWcEZLM05oVVdJNFVEQlZRemRDTWt4TFUzUUtabmt6ZG5oVWRVMHhlRVpsV2t3emVHUllXRVZzWlVWTFVXSlBVM1JXY2tJMlVXVjRkRTVyY0dsM1MwTlVTVlJOUTJFM0wwazVlRzhyVVhBMFVqZHJiUXBJYmxwdlRFNVdkbEZtUkZoTFF6VnNNakpKYjNjclVIcHRaMmRtV1ZNNE1qUkhaVmRJUldKdWFtRkRiRE5pT0ZSb2FuYzlDaTB0TFMwdFJVNUVJRkpUUVNCUVVrbFdRVlJGSUV0RldTMHRMUzB0Q2c9PQo=",
```

Comment:
Does this base64-encoded data is sensitive or not?

Reply:
The unit test data was all from the local temporary KIND environment.